clamd_test.go

```diff
@@ -460,3 +493,17 @@ func copyFile(src, dst string, mode os.FileMode) error {
 	}
 	return os.Chmod(dst, mode)
 }
+
+type eofReader struct {
+	source io.Reader
+	size   int64
+}
+
+func (e *eofReader) Read(p []byte) (n int, err error) {
+	n, err = e.source.Read(p)
```

Comment:
Handle the error that could be returned from e.source.Read(p) by returning it immediately ? Could be better coded as an if block. if n, err = e.source.Read(p); err != nil { return }

Reply:
Done.